src/ovp_pipeline/commands/_ui_renderers.py

```
@@ -6717,19 +6747,41 @@ def _render_today_digest_page(payload: dict) -> str:
+                    "white-space:nowrap'>"
+                    f"<span class='muted'>{et}</span> <strong>{subject_html}</strong>"
+                    "</li>"
                 )
-                for s in samples
-            )
             sample_html = (
                 "<div style='margin-top:.6rem;padding-top:.6rem;"
-                "border-top:1px solid var(--border)'>"
-                f"<ul class='list-tight tiny'>{items}</ul></div>"
+                "border-top:1px solid var(--border);max-width:100%'>"
+                f"<ul class='list-tight tiny' "
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)